.github/workflows/compressed-size.yml

```
@@ -24,25 +27,27 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.pull_request.draft == false
     steps:
+      - uses: styfle/cancel-workflow-action
```

We should include the version number in the reference here. [GitHub strongly recommends against using actions without a version constraint](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses).
```suggestion
      - uses: styfle/cancel-workflow-action@0.8.0
```

---

Good point 👍🏻